src/style.css

```diff
@@ -1,3 +1,60 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --link: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: rgb(216, 216, 216);
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  width: 100%;
+}
```

Comment:
Be cautious with setting the width to 100% on the '.container'. If there is padding or border, it could cause an overflow. Consider using 'box-sizing: border-box;' to include padding and border in the element's total width and height.